ui/src/app/pages/homepage/data-access-requirements.tsx

```diff
@@ -58,10 +60,34 @@ import { ReactComponent as wearable } from 'assets/icons/DAR/wearable.svg';
 import { TwoFactorAuthModal } from './two-factor-auth-modal';
 
 const styles = reactStyles({
-  headerFlexColumn: {
+  registrationOuterHeader: {
```

Comment:
renamed some things which now only apply to "Initial Registration Mode" (as opposed to the new Annual Renewal Mode)